ncaa_vb_pbp.R

```diff
@@ -131,6 +128,28 @@ ncaa_vb_pbp <- function(game_id) {
         TRUE ~ rally_play_number
       )
     )
+
+  plays <- plays %>%
+    select(-home_action, -away_action) %>%
```

Comment:
you don't need to de-select these columns if you're going to select a list of columns after the fact